### DISCUSSION TITLE
esphome-stream-server-v2 by tube0013

### DISCUSSION THREAD

**Discussion by Hedda** [syssi/esphome-zb-gw03#20] [General]
@syssi FYI, [tube0013](https://github.com/tube0013) has forked oxan's esphome stream server and got it working reliably with ESPHome versions newer than 2021.10:

https://github.com/tube0013/esphome-stream-server-v2

**Comment by syssi** [comment]
Thanks for holding all the threads together! ;-) I will have a look.
